specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/stable/2016-05-01/examples/Operations/List.json

```diff
@@ -2,12 +2,7 @@
     "parameters": {
         "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
         "location": "local",
-        "api-version": "2016-05-01",
-        "reportedStartTime": "2017-08-01T00:00:00.00Z",
-        "reportedEndTime": "2017-08-02T00:00:00.00Z",
-        "aggregationGranularity": "",
-        "subscriberId": "",
-        "continuationToken": ""
```

Comment:
example validation will fail and seems we're not using these parameters. Let me know if these are needed.